### ISSUE TITLE
warnings summary not reflected in html-summary (feature request)

### ISSUE THREAD

**Issue by nicain** [pytest-dev/pytest-html#163]
In addition to test failure, pytest captures a warnings summary and reports this at the end of the test summary.  It would be great if this could be reflected in the html summary as well.

**Comment by gnikonorov** [comment]
Hi @davehunt, is this still something that is relevant? If so I'd like to take a stab at implementing it

**Comment by BeyondEvil** [comment]
Sure, feel free to submit a PR! 👍 @gnikonorov

**Comment by gnikonorov** [comment]
Thank you @BeyondEvil!

**Comment by gnikonorov** [comment]
Still planning on working on this.
I can start once https://github.com/pytest-dev/pytest/pull/7255 is merged

**Comment by nepumuk123** [comment]
Hey @gnikonorov this feature would be great. Are you still planning to do this?

**Comment by gnikonorov** [comment]
Hi @nepumuk123, I am yes. We just need to wait for a version of pytest that contains https://github.com/pytest-dev/pytest/pull/7255 to be released. That should be coming soon. I'll update this issue when it happens

**Comment by nepumuk123** [comment]
Cool, thank you.

**Comment by BeyondEvil** [comment]
I think this is merged and released now @gnikonorov :)

**Comment by gnikonorov** [comment]
I'm going to pick this up again, @BeyondEvil 

The solution would involve making a separate 'warnings summary' table similar to the results table but with only 1 data point, the warnings for a test. It would only show tests that emit warnings and would be hidden from view if there are no warnings.

This would involve a merge into your next-gen branch. Do you have any opposition?

**Comment by BeyondEvil** [comment]
> This would involve a merge into your next-gen branch. Do you have any opposition?

I have no idea what the implications will be, but we'll worry about that later. :)
